app/models/account_suggestions/similar_profiles_source.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class AccountSuggestions::SimilarProfilesSource < AccountSuggestions::Source
+  class QueryBuilder < AccountSearchService::QueryBuilder
+    def must_clauses
+      [
+        {
+          more_like_this: {
+            fields: %w(text text.stemmed),
+            like: @account.active_relationships.recent.limit(5).pluck(:target_account_id).map { |id| { _index: 'accounts', _id: id } },
+          },
+        },
+
+        {
+          term: {
+            properties: 'discoverable',
+          },
+        },
+      ]
+    end
+
+    def must_not_clauses
+      [
+        {
+          terms: {
+            id: following_ids,
+          },
+        },
+
+        {
+          term: {
+            properties: 'bot',
+          },
+        },
+      ]
+    end
```

Comment:
`must_not_clauses` is actually not used anywhere in this class.

Reply:
(My bad, I overlooked the changes in `AccountSearchService::Query`, this has been addressed)